tests/ctst/features/azureArchive.feature

```diff
@@ -121,6 +121,38 @@ Feature: Azure Archive
         |               Versioned |           2 |      30000 |               1 |
         |               Suspended |           2 |      30000 |               1 |
 
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    @ColdStorage
+    Scenario Outline: Adjust restore duration on restored object
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-azure-archive"
+    And object "obj-2" should be "transitioned" and have the storage class "e2e-azure-archive"
+    When i restore object "obj-1" for <restoreDays> days
+    And i restore object "obj-2" for <restoreDays> days
+    Then blob for object "obj-1" must be rehydrated
+    And blob for object "obj-2" must be rehydrated
+    Then object "obj-1" should be "restored" and have the storage class "e2e-azure-archive"
+    And object "obj-2" should be "restored" and have the storage class "e2e-azure-archive"
+    And object "obj-1" should expire in <restoreDays> days
+    And object "obj-2" should expire in <restoreDays> days
+    Then i adjust restore duration of object "obj-1" for 50 days
+    And i adjust restore duration of object "obj-2" for 5 days
+    When i wait for 5 days
+    Then object "obj-2" should be "cold" and have the storage class "e2e-azure-archive"
+    And object "obj1" should expire in 45 days
+    When i wait for 45 days
+    Then object "obj-1" should be "cold" and have the storage class "e2e-azure-archive"
+
+    Examples:
+        | versioningConfiguration | objectCount | objectSize | restoreDays |
+        |           Non versioned |           2 |        100 |           15 |
+        |               Versioned |           2 |        100 |           15 |
+        |               Suspended |           2 |        100 |           15 |
```

Comment:
as per the ticket, should also add a test case `restore→expiration→re-restore→expiration`